### PR TITLE
Added exception handling for placing a marker on the board.

### PR DESCRIPTION
**Pull request by ericdrosado**
screen shot 2017-12-11 at 1 04 48 pm

### CODE REVIEW

File: TicTacToe/TicTacToe/UI.cs [hunk 1]
```diff
@@ -22,10 +22,18 @@ public class UI {
...
-            BoardView();
+            try {
+                string input = this.io.GetInput();
+                int move = Int32.Parse(input);
+                this.board.UpdateBoard(move);
+                BoardView();
+            } catch(FormatException) {
+                this.io.Print(InvalidEntryPrompt());
+                PlaceMarker();
+            } catch(IndexOutOfRangeException) {
```
Comment (cdemyanovich) [inline TicTacToe/TicTacToe/UI.cs]: While wondering whether Java could handle multiple exceptions in one `catch`, I found the answer to be yes for Java 7 and later: https://docs.oracle.com/javase/tutorial/essential/exceptions/catch.html. Note sure if this has changed in Java 9, though.

Comment (ericdrosado) [reply TicTacToe/TicTacToe/UI.cs]: Nice! I was able to find a c# equivalent.

Comment (cdemyanovich) [reply TicTacToe/TicTacToe/UI.cs]: That's good, since I forgot this was C#! :-)

**Comment by cdemyanovich** [comment]
If a player provides bad input often enough, it looks like the board will scroll off the screen. Something to keep in mind in terms of usability.

**Pull request merged**